packages/salesforcedx-vscode-replay-debugger/package.json

```diff
@@ -89,6 +104,12 @@
             "view == sfdx.force.view.checkpoint && sfdx:project_opened && sfdx:replay_debugger_checkpoints_enabled",
           "group": "navigation"
         }
+      ],
+      "editor/context": [
```

Comment:
For completeness, should we also make this show up in [explorer/menu](https://code.visualstudio.com/docs/extensionAPI/extension-points#_contributesmenus) when the user is selecting a .log file?
Or do we not have enough information to properly determine the langId of a file before it's opened?

Reply:
I can add to `explorer/menu`. `resourceLangId` looks feasible.